web-common/src/features/dashboards/time-dimension-details/TDDTable.svelte

```diff
@@ -99,7 +100,7 @@
     classesToAdd.push(
       getClassForCell(
         palette,
-        rowIdxHover,
+        rowIdxHover ?? highlightedRow,
```

Comment:
**Security Issue**: The use of nullish coalescing operator with 'rowIdxHover' and 'highlightedRow' could lead to unexpected behavior if 'rowIdxHover' is intentionally set to zero. <br> **Fix**: Adjust the logic to explicitly check for undefined instead of falsy values. <br> **Code Suggestion**: 
 ```
 +        rowIdxHover !== undefined ? rowIdxHover : highlightedRow,
 +        colIdxHover ?? highlightedCol,
 ```